source/rails-girls/_js.haml

```diff
@@ -0,0 +1,13 @@
+:javascript
+  !function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');
+
+
+.fb-root
+  :javascript
+    (function(d, s, id) {
+    var js, fjs = d.getElementsByTagName(s)[0];
+    if (d.getElementById(id)) return;
+    js = d.createElement(s); js.id = id;
+    js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.6&appId=603249739823141";
+    fjs.parentNode.insertBefore(js, fjs);
+    }(document, 'script', 'facebook-jssdk'));
```

Comment:
Had no idea that these were bad. I will get rid of them. There are already a couple of links to fb and twitter. I am ok if you want to add some other links
![social](https://cloud.githubusercontent.com/assets/17249449/16415018/61e6951c-3d3b-11e6-812e-bdc79833c83d.png)